app/handlers/web/auth.go

```diff
@@ -92,10 +92,18 @@ func AuthGithubCallbackHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	var jwtToken string
+	var jwtError error
```

Comment:
@jlaswell looks like that needs to be there in case it doesn't make it through either of the if blocks, the vars are still initialized and can be checked at the bottom.